src/components/TextField/TextField.jsx

```diff
@@ -77,10 +77,17 @@ const TextField = React.createClass(createLucidComponentDefinition({
 		onChangeDebounced: func,
 
 		/**
+		 * Fires an event on every keydown
+		 *
+		 * Signature: `(value, { event, props }) => {}`
+		 */
+		onKeyDown: func,
+
+		/**
 		 * Fires an event when the user hits "enter" from the TextField. You
 		 * shouldn't use it if you're using `isMultiLine`.
 		 *
-		 * Signature: `(value, { event, props }) => {}`
+		 * Signature: `(event) => {}`
```

Comment:
what do you mean? keydown appears to only send the event: https://github.com/appnexus/lucid/blob/appnexus/ANXR-372-export-default-stateful/src/components/TextField/TextField.jsx#L199

tho we should probably change that so it's consistent with the others